.travis.yml

```diff
@@ -0,0 +1,21 @@
+# Configurations
+dist: bionic
+language: python
+python:
+  - "3.9.5"
+
+install: 
+  - pyenv global 3.9.5
+  - pip3 install -U pip wheel setuptools
+  - pip3 install cloudsmith-cli poetry==1.2.0
+
+before_deploy:
```

Comment:
could be that travis does not like the fact that there is no `script` target

can you replace `before_deploy` with `script` and see?